README.md

```diff
@@ -544,6 +547,33 @@ collect(['name' => 'foo'])->none(function ($key, $value) {
 }); // returns true
 ```
 
+### `containsAll`
+
+Checks whether a collection contains all values of the items that are identified.
+
+```php
+collect(['foo', 'bar', 'baz'])->containsAll(['bar', 'foo']); // returns true
+collect(['foo', 'bar'])->containsAll(['bar', 'foo', 'baz']); // returns false
+```
+
+### `containsAny`
+
+Checks whether a collection contains any values of the items that are identified.
+
+```php
+collect(['foo'])->containsAny(['foo', 'baz']); // returns true
+collect(['foo'])->containsAny(['bar', 'baz']); // returns false
+```
+
+### `containsAnyNone`
```

Comment:
I think we can make this one better:

Could you rename this on to `doesNotContain`? 

These should all work:

- `doesNotContain($value)`
- `doesNotContain($value, $anotherValue)`
- `doesNotContain([$value, $anotherValue])` // does same as above

Reply:
Do you want to rename `containsAnyNone ` to `doesNotContain` ?

Reply:
Yup 👍 

I think that sounds more natural.